tools/rum/test/cruncher.test.js

```diff
@@ -768,6 +768,8 @@ describe('DataChunks.hasConversion', () => {
       checkpoint: ['top'],
```

Comment:
What if we write the test something like

```
const spec = {
      facetOne: ['top'],
      facetTwo: ['hidden'],
    };
    d.addFacet('facetOne', (bundle) => bundle.events.map((e) => e.checkpoint));
    d.addFacet('facetTwo', (bundle) => bundle.events.map((e) => e.target));

```

This way it would be more clear that conversions work with facets not bundle events.

Reply:
Good idea.